porady_odnosnie_studiowania.md

```diff
@@ -348,3 +348,93 @@ Choć w poprzednim punkcie dość dużo wagi położyliśmy na to, żeby nie ba
 
     Przepisywanie treści wykładu słowo w słowo nie jest najlepszą taktyką. Znacznie lepiej sprawdza się notowanie zwięzłe, tak, aby móc skupić się na zrozumieniu materiału, zamiast na idealnym skopiowaniu słów prowadzącego. Znajdź balans pomiędzy dokładnością notatek a skupieniem na wykładzie.
   </details>
+
+---
+
+## Praca dyplomowa
+
+* **Wybierz interesujący Cię temat**
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Pracę dyplomową najlepiej pisać na temat na którym się znasz, który Cię interesuje i z którym wiążesz swoją przyszłość. Przede pomoże Ci to pracować nad nią systematycznie i przemóc ewentualną prokrastynację. Ponadto najpewniej zamieścisz tytuł i opis pracy (a może nawet repozytorium pod którym można znaleźć jej treść/kod napisany w jej ramach) w swoim CV/portfolio/liście intencyjnym/etc. - dlatego warto, żeby praca była najwyższej możliwej jakości w czym chęci do tworzenia jej zdecydowanie pomagają.
+  </details>
+
+* **Pracuj systematycznie i zacznij wcześnie**
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Jakkolwiek kuszące może być odłożenie robienia pracy dyplomowej na ostatnią chwilę, jest to bardzo zła praktyka która źle odbije się na jej jakości, Twoim samopoczuciu i finalnej ocenie. Najlepszą strategią jest regularna pisanie pracy i jej komponentów, począwszy od początku semestru w który musisz ją oddać. Temat pracy i jej promotora warto wybrać jeszcze wcześniej.
+  </details>
+
+* **Promotor nie napisze za Ciebie pracy**
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Promotor w procesie pisania pracy dyplomowej jest przewodnikiem oraz oceniającym a nie jednym z jej autorów. Możesz zwrócić się do niego z pytaniami dotyczącymi np. technologii które chcesz wykorzystać czy propozycji rozwiązania/obejścia problemu na który się natknąłeś. Nie licz natomiast, że promotor będzie prowadził Cię za rękę lub pilnował czy pracujesz systematycznie.
+
+    Ponadto powinieneś pamiętać, że Twój promotor najpewniej zajmuje się kilkoma pracami dyplomwymi naraz i każdej musi poświęcić uwagę.
+  </details>
+
+* **Pamiętaj o terminach**
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Praca dyplomowa musi zostać wprowadzona do systemu APD (Archiwum Prac Dyplomowych) na minimum tydzień przed terminem prezentacji/obrony. Jednakże w systemie musi zostać zatwierdzona przez promotora a potem przepuszczona przez system antyplagiatowy (który potrafi sprawdzać pracę nawet do 2-3 dni), dlatego warto zamieścić ją w APD kilka dni przed obowiązkowym terminem.
```

Comment:
Czy nie ma już jakiegoś wymogu donoszenia pracy w wersji papierowej? Kiedyś to trzeba było robić.

Reply:
W tym roku zniesiono konieczność przedstawiania prac dyplomowych w wersji papierowej (na pewno lic. i inż., mgr. chyba też)